Makefile

```diff
@@ -177,9 +177,8 @@ stop:
 compile-pot: .state/env/pyvenv.cfg
 	PYTHONPATH=$(PWD) $(BINDIR)/pybabel extract \
 		-F babel.cfg \
-		--copyright-holder="PyPA" \
-		--msgid-bugs-address="https://github.com/pypa/warehouse/issues/new" \
-		--project="Warehouse" \
+		--omit-header \
+		--no-location \
```

Comment:
I think it'd be a good idea to maybe have separate targets just for identifying if regeneration of `messages.pot` is needed and keep the original one as is.

Reply:
Location info is actually useful as it allows us to track down places in the templates if the context is unclear.